### PR TITLE
Origami cascade

### PR DESCRIPTION
**Pull request by GlynnPhillips**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -18,19 +18,16 @@
 		"tests"
 	],
 	"dependencies": {
-		"alphaville-ui": "Financial-Times/alphaville-ui#^2.6.0",
-		"dom-delegate": "ftdomdelegate#^2.0.0",
+		"alphaville-ui": "Financial-Times/alphaville-ui#fb1f866",
```
Comment (notlee) [inline bower.json]: For when [this is released](https://github.com/Financial-Times/alphaville-ui/pull/18):
```suggestion
		"alphaville-ui": "^4.0.0",
```

**Comment by chee** [comment]
..!
diff: plus 16,266 minus 307

... let's go

File: assets/js/lib/LongroomFormInput.js [hunk 2]
```diff
@@ -111,7 +111,7 @@ function LongroomFormInput (config) {
 		pollForTinymce().then(() => {
 			tinymce.init({
 				selector: `#${input.id}`,
-				height: 300,
+				height: 200,
```
Comment (chee) [inline assets/js/lib/LongroomFormInput.js]: why is this smaller?

File: assets/js/lib/faq.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const Delegate = require('dom-delegate');
+const Delegate = require('ftdomdelegate');
 const dom = require('o-dom');
 
 var selectedIndex = null;
```
Comment (chee) [inline assets/js/lib/faq.js]: why are these vars?

File: assets/js/lib/LongroomFormInput.js (hunk 2 above)
Comment (notlee) [reply assets/js/lib/LongroomFormInput.js]: Glynn fortunately wrote a nice commit:

>Reduce the height of the description text area
>
>As this is no longer hidden by o-expander design felt it was best to
reduce its height slightly

File: assets/js/lib/faq.js (hunk 3 above)
Comment (notlee) [reply assets/js/lib/faq.js]: alphaville  build tools handle the default import. Except where there are non-default exports, which I guess we haven't come across in alphaville-longroom. I'm inclined to leave it

**Pull request merged**